src/core/mesh/qgsmeshdataprovidertemporalcapabilities.h

```diff
@@ -34,21 +34,37 @@ class CORE_EXPORT QgsMeshDataProviderTemporalCapabilities: public QgsDataProvide
 {
   public:
 
+    /**
+     * Method to use when requesting a tempral dataset from a time
```

Comment:
typo: temporal dataset

Reply:
I would clarify the docs, for example: Method for selection of mesh dataset index for time interval from temporal framework